shared/js/otSdkDriver.js

```diff
@@ -502,6 +502,8 @@ loop.OTSdkDriver = (function() {
           break;
         case "Session.networkDisconnected":
         case "Session.forceDisconnected":
+        case "Session.subscribeCompleted":
+        case "Session.screen.subscribeCompleted":
```

Comment:
Why aren't we keeping metrics on (all four of) these?  Because they're logged in some other way?  Probably worth a comment, at the very least.